shm/tests/NextDayTest.cpp

```diff
@@ -67,7 +67,6 @@ TEST_F(NextDayTest, NextDayShouldChangeStockInStore) {
 
 TEST_F(NextDayTest, NextDayShouldPayCrew) {
     auto player_money = player.getMoney();
-    Ship ship = Ship(30, 10, 1, &player);
```

Comment:
why did you remove this line?

Reply:
Accidentally - but please look that we already have identical `Ship` in fixture - ship(30, 10, 1, &player) {}, so it's event better ;d

Reply:
You are right